### PR TITLE
chore: version bump

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

Version bump. I'm skipping 0.18.1 since I created a tag pointing to the wrong version. This one will get a new tag and a new package.json version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unleash/proxy",
-  "version": "0.18.0",
+  "version": "0.18.2",
```
Comment (sighphyre) [inline package.json:3]: Oh! Any idea what happened to 0.18.1?

**Review by sighphyre — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (kwasniew) [reply package.json:3]: I created incorrect tag 0.18.1 pointing to 0.18.0 without bumping package.json. I prefer not to mutate anything and roll forward instead.

Comment (sighphyre) [reply package.json:3]: Gotcha! No worries, was just curious

**Pull request merged**